src/index.js

```diff
@@ -1,13 +1,126 @@
-export default function BaseballGame() {
-  this.play = function (computerInputNumbers, userInputNumbers) {
-    return "결과 값 String";
-  };
-}
-
-// export default class BaseballGame {
-//   play(computerInputNumbers, userInputNumbers) {
-//     return "결과 값 String";
-//   }
-// }
-
-new BaseballGame();
+export default class BaseballGame { 
+
+    constructor() {
+      this.computerNum = this.randomNumGenerator();
+      this.strike = 0;
+      this.ball = 0;
+      this.result = '';
+      console.log(this.computerNum);
+    }
+  
+    randomNumGenerator() {
+      let first = Math.floor(Math.random() * 9 + 1);
+      let second = Math.floor(Math.random() * 9 + 1);
+      while (first == second) {
+        second = Math.floor(Math.random() * 9 + 1);
+      }
+      let third = Math.floor(Math.random() * 9 + 1);
+      while (third == first || third == second) {
+        third = Math.floor(Math.random() * 9 + 1);
+      }
+      return (
+        third.toString() + second.toString() + first.toString()
```

Comment:
```suggestion
        third.toString() + second.toString() + first.toString()
```
아래와 같은 로직으로 짜면 조금 더 간단해질 수 있을것같은데 .toString()과 array의 비용은 따로 찾아봐야할것같습니다.
```
let arr = [];
arr.push(Math.floor(Math.random() * 9 + 1));
...

arr.join('');
```